### ISSUE TITLE
ReferenceError: The verison of babel-runtime of undefined that you have installed does not match the babel verison of 5.1.9

### ISSUE THREAD

**Issue by assaf** [babel/babel#1266]
From what I could tell gather, this happens when installing an older version of babel (e.g. 5.1.8), which will then install the latest babel-core (e.g. 5.1.9) and then complain about a mismatch.

```
$ npm ls | grep babel
├─┬ babel@5.1.7
│ ├─┬ babel-core@5.1.9
├─┬ babel-runtime@5.1.7
│ ├─┬ babel-core@5.0.12
│ │ ├─┬ regenerator-babel@0.8.13-2
```

**Comment by sebmck** [comment]
I don't know why you've got a newer version of `babel-core` installed in `babel`. The runtime version check can probably be removed since it's no longer relevant as 5.x is going to try and guarantee runtime/helper stability.

**Comment by assaf** [comment]
The dependency specification is `"babel-core": "^5.1.7",` npm will load the most recent version that matches, hence 5.1.9.

**Comment by sebmck** [comment]
I know, but installing `babel` should also get you the latest.

**Comment by assaf** [comment]
Occasionally things break, e.g https://github.com/babel/babel/issues/1265 Using `@latest` is too painful, so I'm upgrading weekly and in-between running against a known working version.

**Comment by sebmck** [comment]
Fixed as of 5.1.10.

**Comment by assaf** [comment]
Sweet, thanks!
